tasks/src/fill-mask/about.md

```diff
@@ -29,11 +33,13 @@ Would you like to learn more about the topic? Awesome! Here you can find some cu
 - [Workshop on Pretraining Language Models and CodeParrot](https://www.youtube.com/watch?v=ExUR7w6xe94)
 
 ### Notebooks
+
```

Comment:
Same as above, I'd rather have no gaps

Reply:
It does not affect rendering. Might help for for readability :D